CMakeLists.txt

```diff
@@ -0,0 +1,344 @@
+#[===========================================================================[
+AUTHOR(S):  Rashad Kanavath <rashad km gmail>
+PURPOSE:    GRASS GIS root that adds options to activate/deactivate
+            3rd party libraries
+COPYRIGHT:  (C) 2020-2022 by the GRASS Development Team
+            This program is free software under the GPL (>=v2)
+            Read the file COPYING that comes with GRASS for details.
+#]===========================================================================]
```

Comment:
Please update as needed and remove the ASCII graphics around the comment unless there is some CMake reason for it.

Reply:
Perhaps we should insert a `SPDX-License-Identifier: GPL-2.0-or-later` (see #4190) to all new files.

Reply:
I now see different styles throughout. I really don't know what is expected, but these ASCII lines are not as popular as they once was, so I would say, keep only what is syntactically necessary.